docs/faq.md

```diff
@@ -1,24 +0,0 @@
-<!-- omit in toc -->
-
-# FAQ
-
-- [Would it be possible to host the application page and Oauth api on different domains?](#would-it-be-possible-to-host-the-application-page-and-oauth-api-on-different-domains)
```

Comment:
The whole question is not relevant anymore, so Nikola and I decided to remove it.